Infrastructure.DataAccess/Migrations/SQLScripts/Migrate_Users_Not_Associated_With_Any_Org.sql

```diff
@@ -0,0 +1,26 @@
+﻿/*
+User story reference: 
+    https://os2web.atlassian.net/browse/KITOSUDV-3422
+ 
+Content:
+    Sets Users with no assossiated Organizations as deleted
+*/
+
+BEGIN
+	UPDATE [User]
+	SET 
+		Name = 'Slettet bruger',
+		LockedOutDate = GETDATE(),
+		Email = CONVERT(NVARCHAR(36), NEWID()) + '_deleted_user@kitos.dk',
```

Comment:
I think we should also do what the "delete command" does (in code) which covers:
- Remove all sso identities
- Remove all system roles
- Remove all contract roles
- Remove all dpr roles
- Remove all org unit roles